javaparser-core-generators/src/main/java/com/github/javaparser/generator/core/other/TokenKindGenerator.java

```diff
@@ -60,7 +60,7 @@ public TokenKindGenerator(SourceRoot sourceRoot, SourceRoot generatedJavaCcSourc
         final EnumDeclaration kindEnum = javaToken.findFirst(EnumDeclaration.class, e -> e.getNameAsString().equals("Kind")).orElseThrow(() -> new AssertionError("Can't find class in java file."));
 
         kindEnum.getEntries().clear();
-        this.annotateGenerated(kindEnum);
+        AbstractGenerator.annotateGenerated(kindEnum, this.getClass().getName());
```

Comment:
Yeah, those methods should be moved out of AbstractGenerator, or this one should become an AbstractGenerator.